app.js

```diff
@@ -0,0 +1,157 @@
+//jshint esversion: 6
```

Comment:
Too much responsibility given to a single file. The problem with that are follows:
1. If I ask you to make another API then you might feel to divide the file, in sevaral other files.
2. When your base design is week, you may produce the initial thing faster but may take more time when changes come because you will tend to refactor more and may affect the parts of the code that should not have ideally been affected.
3. Thus it may lead to more bugs.
4. You have used a functional approach you could have instead made classes in node js, something like DatabaseManager, RedisManager, Routes, StudentAPI etc.